### DISCUSSION TITLE
Entry in ltconfig.conf to set all torrents to paused or stopped state

### DISCUSSION THREAD

**Discussion by liquid-ion** [arvidn/libtorrent#7655] [Q&A]
Wondering if there is a way I can set a parameter in ltconfig.conf to set all torrents to "Stopped" or "Paused" status on startup.

Mainly I don't want any actively downloading torrents to continue downloading.

**Comment by arvidn** [comment]
you can set the [paused](https://libtorrent.org/reference-Core.html#torrent_flags_t) flag in the [add_torrent_params](https://libtorrent.org/reference-Add_Torrent.html#add_torrent_params) when adding the torrents. Presumably, on startup, you load the torrents from disk into `add_torrent_params` objects. You can modify the flags to set them all to paused before passing them on to `session::add_torrent()`.

If you really want a *setting* to have this effect, you could set [active_limit](https://libtorrent.org/reference-Settings.html#active_limit) to 0.
